CONTRIBUTING.md

```diff
@@ -5,56 +5,107 @@ When contributing to this repository, please describe the change you wish to mak
 Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
 ## Setting Up Development Environment
-Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools. 
-### Prerequisites
+
+Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools.
+
+### With Devbox
+
+The easiest way to setup and build devbox is with Devbox!
+
+* Install Devbox using the command below. If you don't have Nix installed, Devbox will automatically install it for you when you run a command.
+
+```bash
+curl -fsSL https://get.jetpack.io/devbox | bash
+```
+
+* Start a Devbox Shell for the project
+
+```bash
+devbox shell
+```
+
+* Build the Devbox CLI:
+
+```bash
+# Build for your current OS
+devbox run build
+
+# Build for Linux
+devbox run build-linux
+
+# Run the CI Linter
+devbox run lint
+```
+
+* Run and Test the Devbox CLI:
+
+```bash
+./dist/devbox <test_command>
+```
+
+* For the best experience working on Devbox with VSCode, we recommend running:
+
+```bash
+devbox shell -- code
```

Comment:
We may need to change this to `devbox run code` after the feature flag for devbox run is removed.

Reply:
Sounsd good, is the command right for the current build?

Reply:
Yes, it's just behind a feature flag

Reply:
Actually, this is tricky because it won't work both inside or outside the shell. Maybe the easiest option is to add a script to our devbox.json